examples/pspnet/caffe2npz.py

```diff
@@ -0,0 +1,251 @@
+from __future__ import print_function
+
+import argparse
+from google.protobuf import text_format
+import numpy as np
+import os
+import re
+
+import chainer
+import chainer.links as L
+from chainer import serializers
+from chainercv.experimental.links import PSPNetResNet101
+
+import caffe_pb2
+
+
+def get_chainer_model(n_class, input_size, n_blocks, pyramids, mid_stride):
+    with chainer.using_config('train', True):
+        model = PSPNetResNet101(
+            n_class, None, input_size)
+        model(np.random.rand(1, 3, input_size, input_size).astype(np.float32))
+    size = 0
+    for param in model.params():
+        try:
+            size += param.size
+        except Exception as e:
+            print(str(type(e)), e, param, param.name)
+            exit(-1)
```

Comment:
Why do you catch an exception? Usually, Python interpreter does the similar thing without explicit catching.